specifications/1614618-timelines.md

```diff
@@ -0,0 +1,95 @@
+- Feature Name: Timelines
+- Bug: [bug 1614618](https://bugzilla.mozilla.org/show_bug.cgi?id=1614618)
+
+# Summary
+
+Add Timelines to Projects to educate localizers on future milestones on a
+project that's continuously shipping.
+
+# Motivation
+
+Pontoon currently supports Deadlines. They're one-off points in time, and they
+pose a few limitations:
+
+* Deadline sounds permanent, threatening, and ugly.
+* Project managers can't set more than one.
+* In practice, many deadlines are in the past.
+* Most projects at Mozilla continue to ship past a deadline.
+
+# Roles
+
+| Role | Impact |
+| -- | -- |
+| Localizer | Only see upcoming points in time |
+| L10n Project Manager (LPM) | Set multiple points in time for a project |
+
+# New Roles
+
+| Role | Impact |
+| -- | -- |
+| Project Manager (PPM) | PPM can set multiple points in time for a project|
+
+This feature introduces a role of the upstream project manager. In contrast to
+the Localization Project Manager (LPM), this is a Project Project Manager, thus
+we're introducing the acronym PPM.
```

Comment:
I like this section!

I think adding such role is a separate feature, though:
- I don't see the need for it in the Motivation section.
- We should let PPMs edit other project details.
- I'd even let them [verify/submit projects](https://bugzilla.mozilla.org/show_bug.cgi?id=1204659) to be enabled on Pontoon.

Reply:
I don't understand this PPM / LPM thing at all. I agree that it seems good to have a section about roles though, if only for the benefit of me being confused by this content. :smile: 

I'd suggest merging the two roles-related sections into one. And I think we should also consolidate the list of roles in Pontoon, putting that list in the future template, asking each spec to explain clearly how a feature impacts each role. That means we need to take some time to clarify what the roles currently are in Pontoon, though, because I for one am totally not sure what they are.